### DISCUSSION TITLE
Typo error in Welcome to SWR Discussions! #1684

### DISCUSSION THREAD

**Discussion by nikosantis** [vercel/swr#2068] [General]
Hi.
I was checking how you wrote the docs (.github) for the repo, and I found a small typo error in the `Typo error in Welcome to SWR Discussions! #1684`
https://github.com/vercel/swr/discussions/1684
The URL for the [Code of Conduct](https://github.com/vercel/swr/blob/main/CODE_OF_CONDUCT.md):
Missing the "./github/" folder :)

Bye :)

**Comment by huozhi** [comment]
Updated! Thank you!
